themes/jaeger-docs/layouts/partials/footer.html

```diff
@@ -1,6 +1,6 @@
 {{ $githubRepo    := .Site.Params.githubrepo }}
 {{ $twitterHandle := .Site.Params.twitterhandle }}
-{{ $gitterHandle  := .Site.Params.gitterhandle }}
+{{ $chatHandle  := .Site.Params.chathandle }}
```

Comment:
Var not defined in config and not used here

Reply:
Oops